hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/admin/nssummary/NSSummaryCLIUtils.java

```diff
@@ -0,0 +1,174 @@
+              isSpnegoEnabled);
+      httpURLConnection.connect();
+      int errorCode = httpURLConnection.getResponseCode();
+      InputStream inputStream = httpURLConnection.getInputStream();
+
+      if ((errorCode == HTTP_OK) || (errorCode == HTTP_CREATED)) {
+        return IOUtils.toString(inputStream, StandardCharsets.UTF_8);
+      }
+
+      if (httpURLConnection.getErrorStream() != null) {
+        System.out.println("Recon is being initialized. Please wait a moment");
+        return null;
+      } else {
+        System.out.println("Unexpected null in http payload," +
+            " while processing request");
+      }
+      return null;
+    } catch (ConnectException ex) {
+      System.err.println("Connection Refused. Please make sure the " +
+          "Recon Server has been started.");
+      return null;
+    } catch (AuthenticationException authEx) {
+      System.err.println("Authentication Failed. Please make sure you " +
+          "have login or disable Ozone security settings.");
+      return null;
+    }
+  }
+
+  public static HashMap<String, Object> getResponseMap(String response) {
+    return new Gson().fromJson(response, HashMap.class);
+  }
+
+  public static void printNewLines(int cnt) {
+    for (int i = 0; i < cnt; ++i) {
+      System.out.println();
+    }
+  }
+
+  public static void printSpaces(int cnt) {
+    for (int i = 0; i < cnt; ++i) {
+      System.out.print(" ");
+    }
+  }
+
+  public static void printEmptyPathRequest() {
+    System.err.println("The path parameter is empty.\n" +
+        "If you mean the root path, use / instead.");
+  }
+
+  public static void printPathNotFound() {
+    System.err.println("Path not found in the system.\n" +
+        "Did you remove any protocol prefix before the path?");
+  }
+
+  public static void printTypeNA(String requestType) {
+    String markUp = "@|underline " + requestType + "|@";
+    System.err.println("Path found in the system.\nBut the entity type " +
+        "is not applicable to the " + Ansi.AUTO.string(markUp) + " request");
+  }
+
+  public static void printKVSeparator() {
+    System.out.print(" : ");
+  }
+
+  public static void printWithUnderline(String str, boolean newLine) {
+    String markupStr = "@|underline " + str + "|@";
+    if (newLine) {
+      System.out.println(Ansi.AUTO.string(markupStr));
+    } else {
+      System.out.print(Ansi.AUTO.string(markupStr));
+    }
+  }
+
+  public static void printFSOReminder() {
+    printNewLines(1);
+    System.out.println("[Warning] FSO is NOT enabled. " +
+        "Namespace CLI is only designed for FSO mode.\n" +
+        "To enable FSO set ozone.om.enable.filesystem.paths to true " +
+        "and ozone.om.metadata.layout to PREFIX.");
+    printNewLines(1);
+  }
+
+  public static String parseInputPath(String path) {
+    if (!path.startsWith("ofs://")) {
+      return path;
+    }
+    int idx = path.indexOf("/", OFS_PREFIX.length());
```

Comment:
ofs:// path takes in OM host:port or service id as the first part after the ofs:// prefix. Hence, this code needs to handle that as well.